prov/verbs/src/fi_verbs.c

```diff
@@ -43,28 +43,20 @@ static const char *local_node = "localhost";
 
 #define VERBS_DEFAULT_MIN_RNR_TIMER 12
 
-size_t verbs_default_tx_size 		= 384;
-size_t verbs_default_rx_size 		= 384;
-size_t verbs_default_tx_iov_limit 	= 4;
-size_t verbs_default_rx_iov_limit 	= 4;
-size_t verbs_default_inline_size 	= 64;
-
-size_t verbs_min_rnr_timer = VERBS_DEFAULT_MIN_RNR_TIMER;
-
 struct fi_ibv_gl_data fi_ibv_gl_data = {
 	.def_tx_size		= 384,
 	.def_rx_size		= 384,
 	.def_tx_iov_limit	= 4,
 	.def_rx_iov_limit	= 4,
-	.min_rnr_timer          = VERBS_DEFAULT_MIN_RNR_TIMER,
+	.min_rnr_timer		= VERBS_DEFAULT_MIN_RNR_TIMER,
 	.fork_unsafe		= 0,
 	.use_odp		= 1,
 	.cqread_bunch_size	= 8,
 	.iface			= "ib",
 
 	.rdm			= {
 		.buffer_num		= FI_IBV_RDM_TAGGED_DFLT_BUFFER_NUM,
-		.buffer_size		= FI_IBV_RDM_DFLT_BUFFER_SIZE,
+		.buffer_size		= FI_IBV_RDM_DFLT_BUFFERED_SIZE,
```

Comment:
Please move fix into first commit, so we don't have a broken build at that point in time.

Reply:
Yes, agree. will do that